branded_ios_app/publishing_ios_app_7.rst

```diff
@@ -69,7 +69,9 @@ on the app. The default is 30.
 
 The next section is for uploading your custom artwork to be built 
 into the app. The ownBuilder app tells you exactly which images you need, and 
-their required size. Click the example images on the right to enlarge them.
+their required size. You only need one Splash Screen image, and ownBrander 
+will automatically resize and crop it for different-sized screens. Click the 
```

Comment:
I'd add that not only the image is needed but also the background color for the splash screen.
FYI, this is used to ensure the ratio of the image so that the image looks nicely no matter the device dimensions